plugins/mcreator-localization/lang/texts.properties

```diff
@@ -170,6 +170,10 @@ dialog.gui.label_event_display_condition=Label display condition
 dialog.gui.label_text=Text: 
 dialog.gui.label_component_title=Label editor
 dialog.gui.label_text_color=Text color:
+dialog.gui.add_entity_model=Model editor
+dialog.gui.entity_model_condition=Display model from entity
```

Comment:
This one is not really a condition (does not return logic value), I'd use the `_procedure` prefix for the key.